templates/object/12/Classes/Domain/Model/objectNamePascal.php

```diff
@@ -14,4 +14,16 @@ class objectNamePascal extends AbstractEntity
      * @var string
      */
     public string $title = '';
+
+    ######################
+    ## Helper functions ##
+    ######################
+
+    /**
+     * @return string
+     */
+    public function getCode(): string
+    {
+        return md5($this->uid . $GLOBALS['TYPO3_CONF_VARS']['SYS']['encryptionKey']);
```

Comment:
Jeg synes ikke denne skal ind som standard, eftersom det bliver mere og mere sjældent at den skal bruges. **Hvis** den skal ind, skal den laves lidt anderledes eftersom at code er det samme for 2 forskellige objekter med samme uid.